core/key.go

```diff
@@ -132,6 +133,12 @@ type CollectionIndexKey struct {
 
 var _ Key = (*CollectionIndexKey)(nil)
 
+type SchemaVersionKey struct {
```

Comment:
todo: I think it needs a comment here

Reply:
Cheers - I missed that, will add

- [ ] Doc key